8x8 inverted LED displays don't display images or text

I have an 8x8 display and whenever I set it to "INVERTED_PANEL_ROWS": true it throws the following error whenever I call an image display on the panel. The LED panel also displays nothing when this happens

```
2023-11-25 14:06:38.992: RHUtils [ERROR] Exception via catchLogExceptionsWrapper
Traceback (most recent call last):
  File "/home/aaron/RotorHazard/src/server/RHUtils.py", line 192, in wrapper
    return func(*args, **kwargs)
  File "/home/aaron/RotorHazard/src/server/led_event_manager.py", line 140, in activateEffect
    result = args['handler_fn'](args)
  File "/home/aaron/RotorHazard/src/server/plugins/rh_led_handler_bitmap/__init__.py", line 67, in showBitmap
    setPixels(output_img)
  File "/home/aaron/RotorHazard/src/server/plugins/rh_led_handler_bitmap/__init__.py", line 31, in setPixels
    px = img.getpixel((c, row))
  File "/home/aaron/.local/lib/python3.9/site-packages/PIL/Image.py", line 1481, in getpixel
    return self.im.getpixel(xy)
IndexError: image index out of range
```

Seems to work fine on the 16x16 display I have and is also fine if INVERTED_PANEL_ROWS is set to false.

It also does the same thing with text

```
2023-11-25 14:12:22.399: RHUtils [ERROR] Exception via catchLogExceptionsWrapper
Traceback (most recent call last):
  File "/home/aaron/RotorHazard/src/server/RHUtils.py", line 192, in wrapper
    return func(*args, **kwargs)
  File "/home/aaron/RotorHazard/src/server/led_event_manager.py", line 140, in activateEffect
    result = args['handler_fn'](args)
  File "/home/aaron/RotorHazard/src/server/plugins/rh_led_handler_character/__init__.py", line 28, in dataHandler
    printCharacter(args)
  File "/home/aaron/RotorHazard/src/server/plugins/rh_led_handler_character/__init__.py", line 111, in printCharacter
    setPixels(strip, img)
  File "/home/aaron/RotorHazard/src/server/plugins/rh_led_handler_character/__init__.py", line 260, in setPixels
    px = img.getpixel((c, row))
  File "/home/aaron/.local/lib/python3.9/site-packages/PIL/Image.py", line 1481, in getpixel
    return self.im.getpixel(xy)
IndexError: image index out of range
```

The solid colours or flashing ones seem to work ok through

--- a/src/server/plugins/rh_led_handler_bitmap/__init__.py
+++ b/src/server/plugins/rh_led_handler_bitmap/__init__.py
@@ -16,7 +16,7 @@ def showBitmap(args):
     else:
         return False
 
-    def setPixels(img):
+    def setPixels(img, panel_w):
         pos = 0
         for row in range(0, img.height):
             for col in range(0, img.width):
@@ -26,7 +26,7 @@ def showBitmap(args):
                 c = col
                 if Config.LED['INVERTED_PANEL_ROWS']:
                     if row % 2 == 0:
-                        c = 15 - col
+                        c = (panel_w - 1) - col
 
                 px = img.getpixel((c, row))
                 strip.setPixelColor(pos, Color(px[0], px[1], px[2]))
@@ -64,7 +64,7 @@ def showBitmap(args):
             output_img.paste(img, (pad_left, pad_top))
             output_img = output_img.rotate(90 * Config.LED['PANEL_ROTATE'], expand=True)
 
-            setPixels(output_img)
+            setPixels(output_img, panel_w)
             strip.show()
             gevent.sleep(delay/1000.0)
 

--- a/src/server/plugins/rh_led_handler_character/__init__.py
+++ b/src/server/plugins/rh_led_handler_character/__init__.py
@@ -108,7 +108,7 @@ def printCharacter(args):
 
     img = panel['im'].rotate(90 * Config.LED['PANEL_ROTATE'], expand=True)
 
-    setPixels(strip, img)
+    setPixels(strip, img, panel['width'])
     strip.show()
 
 def scrollText(args):
@@ -146,7 +146,7 @@ def scrollText(args):
         panel['draw'].rectangle((0, 0, panel['width'], panel['height']), fill=(0, 0, 0))
         panel['draw'].text((-i, draw_y), text, font=font, fill=(color))
         img = panel['im'].rotate(90 * Config.LED['PANEL_ROTATE'], expand=True)
-        setPixels(strip, img)
+        setPixels(strip, img, panel['width'])
         strip.show()
         gevent.sleep(10/1000.0)
 
@@ -223,7 +223,7 @@ def multiLapGrid(args):
             panel['draw'].text((pos_x + 1, pos_y), text, font=font, fill=color)
 
     img = panel['im'].rotate(90 * Config.LED['PANEL_ROTATE'], expand=True)
-    setPixels(strip, img)
+    setPixels(strip, img, panel['width'])
     strip.show()
 
 def getPanelImg(strip):
@@ -245,7 +245,7 @@ def getPanelImg(strip):
         'draw': ImageDraw.Draw(im)
     }
 
-def setPixels(strip, img):
+def setPixels(strip, img, panel_w):
     pos = 0
     for row in range(0, img.height):
         for col in range(0, img.width):
@@ -255,7 +255,7 @@ def setPixels(strip, img):
             c = col
             if Config.LED['INVERTED_PANEL_ROWS']:
                 if row % 2 == 0:
-                    c = 15 - col
+                    c = (panel_w - 1) - col
 
             px = img.getpixel((c, row))
             strip.setPixelColor(pos, Color(px[0], px[1], px[2]))

